string MedicineName
string Dose
uint8 TimesPerDay
time StartDate
uint8 Duration
string RepeatPrescription
